src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/PlayKmlTours/PlayKmlTours.xaml

```diff
@@ -5,55 +5,31 @@
     <Grid>
         <esriUI:SceneView x:Name="MySceneView" />
         <Border Style="{StaticResource BorderStyle}">
-            <Grid>
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="*" />
-                    <ColumnDefinition Width="*" />
-                    <ColumnDefinition Width="*" />
-                </Grid.ColumnDefinitions>
-                <Grid.RowDefinitions>
-                    <RowDefinition Height="Auto" />
-                    <RowDefinition Height="Auto" />
-                    <RowDefinition Height="Auto" />
-                </Grid.RowDefinitions>
-                <TextBlock Grid.Row="0"
-                           Grid.Column="0"
-                           Grid.ColumnSpan="3"
-                           Margin="0,0,0,5"
-                           FontWeight="SemiBold"
-                           Text="Use the buttons to control the tour. Contains audio. 🎧"
-                           TextWrapping="WrapWholeWords" />
-                <Button x:Name="PlayButton"
-                        Grid.Row="1"
-                        Grid.Column="0"
-                        HorizontalAlignment="Stretch"
-                        Click="Play_Clicked"
-                        Content="Play"
-                        IsEnabled="False" />
-                <Button x:Name="PauseButton"
-                        Grid.Row="1"
-                        Grid.Column="1"
-                        Margin="5,0,5,0"
-                        HorizontalAlignment="Stretch"
-                        Click="Pause_Clicked"
-                        Content="Pause"
-                        IsEnabled="False" />
-                <Button x:Name="ResetButton"
-                        Grid.Row="1"
-                        Grid.Column="2"
-                        HorizontalAlignment="Stretch"
-                        Click="Reset_Clicked"
-                        Content="Reset"
-                        IsEnabled="False" />
+            <StackPanel Orientation="Vertical" Spacing="5">
+                <TextBlock Margin="0,0,0,5" Text="Contains audio. 🎧" />
```

Comment:
I think this label should still be `Use the buttons to control the tour. Contains audio. 🎧` on WPF and WinUI.

Reply:
`Use the buttons to control the tour` is handholdy. I don't think it adds value to the sample.

Reply:
I think since we have space for it in the WPF and WinUI viewers we should keep it in. 

It makes it more welcoming for a user to see some text that accompanies the buttons we have included. It also rules out any uncertainty about what the buttons are doing. In our other samples we include handholdy text to ensure that the users are getting the most out of the sample with minimum effort.